### ISSUE TITLE
Don't run script in parallel! 

### ISSUE THREAD

**Issue by boulund** [boulund/phaster_scripts#1]
With the basic plain text tsv database implementation the script will not work with multiple instances accessing the database at the same time, so don't submit a lot of jobs in parallel! 

To remedy this, I have been thinking about a simple SQLite database instead, but this is fairly low priority right now.

**Comment by boulund** [comment]
A warning note about this was added to the README to replace this issue.
